test/Test/Data/Map.purs

```diff
@@ -65,19 +66,7 @@ instance ordSmallKey :: Ord SmallKey where
   compare = compare `on` smallKeyToInt
 
 instance arbSmallKey :: Arbitrary SmallKey where
-  arbitrary = do
-    n <- arbitrary
-    pure case n of
-      _ | n < 0.1 -> A
-      _ | n < 0.2 -> B
-      _ | n < 0.3 -> C
-      _ | n < 0.4 -> D
-      _ | n < 0.5 -> E
-      _ | n < 0.6 -> F
-      _ | n < 0.7 -> G
-      _ | n < 0.8 -> H
-      _ | n < 0.9 -> I
-      _ -> J
+  arbitrary = oneOf (pure A) (map pure [B, C, D, E, F, G, H, I])
```

Comment:
I think [`elements`](https://pursuit.purescript.org/packages/purescript-quickcheck/3.1.1/docs/Test.QuickCheck.Gen#v:elements) would make this even neater.